packages/proxy/lib/http/request-middleware.ts

```diff
@@ -44,17 +44,15 @@ const MaybeAttachCrossOriginCookies: RequestMiddleware = function () {
     this.req.isAUTFrame,
   )
 
-  const cookies = this.getCookieJar().getCookies(this.req.proxiedUrl, sameSiteContext)
-  const existingCookies = this.req.headers['cookie'] ? [this.req.headers['cookie']] : []
-  const cookiesToAdd = cookies.map((cookie) => `${cookie.key}=${cookie.value}`)
+  const applicableCookiesInCookieJar = this.getCookieJar().getCookies(this.req.proxiedUrl, sameSiteContext)
+  const cookiesOnRequestString = (this.req.headers['cookie'] ? this.req.headers['cookie'] : '').split('; ')
```

Comment:
```suggestion
  const cookiesOnRequest = (this.req.headers['cookie'] || '').split('; )
```

Reply:
addressed in [99d5ed2](https://github.com/cypress-io/cypress/pull/23438/commits/99d5ed23b824ec8206a6424edec54d010c18ec09)